exporters/sender/jdk/src/test/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHttpSenderTest.java

```diff
@@ -63,9 +76,23 @@ void setup() throws IOException, InterruptedException {
                 .build());
   }
 
+  @Test
+  void exportAsJson() {
+    HttpExporter<Marshaler> exporter =
+        new HttpExporterBuilder<>("jdk", "test", server.httpUri().toASCIIString())
+            .exportAsJson()
+            .build();
+
+    server.enqueue(HttpResponse.of(HttpStatus.OK));
+
+    CompletableResultCode result = exporter.export(new NoOpMarshaler(), 0);
+    result.join(1, TimeUnit.MINUTES);
+    Assertions.assertThat(result.isSuccess()).isTrue();
```

Comment:
could we use awaitility for this, rather than result.join?

Something like `await().untilAsserted(() -> Assertions.assertThat(result.isSuccess()).isTrue());`

Reply:
https://github.com/jack-berg/opentelemetry-java/pull/17

Reply:
I think in cases like this where the future is expected to resolve quickly and deterministically, its ok to use join. For example, we use it all over the place in [AbstractHttpTelemetryExporterTest](https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/AbstractHttpTelemetryExporterTest.java).

With that said, I'm actually not sure what this new test has to do with this change so I've gone ahead and removed it.